extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -33,6 +31,7 @@
 
 	&__counter {
 		color: $color-text;
+		padding-bottom: 12px;
```

Comment:
@weronika but language selector has to have padding-bottom: 12px to push dorpdown-content little bit down. In your solution it will cause 24px gap